messageBundlesCompiler.js

```diff
@@ -0,0 +1,198 @@
+/**
+ * The message bundles compiler.
+ *
+ * Contains various functions for compiling a mega message bundle which contains
+ * all messages for all available locales.
+ * Copyright 2017 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ * The research leading to these results has received funding from the European Union's
+ * Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fs = require("fs");
```

Comment:
I appreciate the worry about doing this from within a grunt task, but I think it would be better in the long run to Infusionise this since it is going to end up in one of @the-t-in-rtf 's packages or else the core framework before long. In practice we found that it was fine to simply issue require("infusion") within these tasks since in any case they cannot execute before npm install has finished, since grunt itself needs to be installed. You can retain the existing style of exporting a map of file exports at the base of the file to ease use within Grunt